releasenotes/notes/migrate-algrithms-from-terra-6a8aa994d1206806.yaml

```
@@ -0,0 +1,15 @@
+---
+other:
```

```suggestion
prelude:
```
I think it would be better as prelude (thats the section I mentioned in the issue).